docs/pages/database-access/guides/redis.mdx

```diff
@@ -23,18 +23,76 @@ This guide will help you to:
 
 ## Prerequisites
 
-- Teleport version `9.0` or newer.
 - Redis version `6.0` or newer.
 - `redis-cli` installed and added to your system's `PATH` environment variable.
+- A host where you will run the Teleport Database Service. Teleport must be installed.
+
+  See [Installation](../../installation.mdx) for details.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
 
 <Admonition type="note" title="Note">
   Redis `7.0` and RESP3 (REdis Serialization Protocol) are currently not supported.
 </Admonition>
 
-## Step 1/5. Set up Teleport Auth and Proxy
+## Step 1/5. Install and configure Teleport
+
+### Set up the Teleport Auth and Proxy Services
 
 (!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
+### Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:3080 \
+  --name=example-redis \
+  --protocol=redis \
+  --uri=rediss://redis.example.com:6379 \
```

Comment:
```suggestion
  --uri=redis://redis.example.com:6379 \
```

Reply:
`rediss` is correct (means "redis secure" protocol).